client/README.md

```diff
@@ -2,17 +2,23 @@
 
 ## Requirements
 
-- nodejs 4.2.2
-- running `weavescope` container
 
-## Getting Started
+## Getting Started (using local node)
 
+- You need nodejs 4.2.2 and a running `weavescope` container
 - Setup: `npm install`
 - Build: `npm run build`, output will be in `build/`
 - Develop: `BACKEND_HOST=<dockerhost-ip>:4040 npm start` and then open `http://localhost:4042/`
 
 This will start a webpack-dev-server that serves the UI and proxies API requests to the container.
 
+## Getting Started (using node in a container)
+
+- You need a running `weavescope` container
+- Develop: `make WEBPACK_SERVER_HOST=<dockerhost-ip> client-start` and then open `http://<dockerhost-ip>:4042/`
```

Comment:
From the webpack server's point of view, the backend is running on localhost (the docker host). From the browser's point of view the docker host may be a docker machine or dlite, and localhost is the Mac.